payments/utils.py

```diff
@@ -414,12 +417,12 @@ def send_refund_notice(order):
 def prepare_for_resending(order):
     if order.status == OrderStatus.ERROR or order.lease.status == LeaseStatus.ERROR:
         order.set_status(
-            OrderStatus.WAITING,
+            OrderStatus.OFFERED,
             comment=f"{today()}: {_('Cleanup the invoice to attempt resending')}\n",
         )
     elif order.status == OrderStatus.CANCELLED:
         order.set_status(
-            OrderStatus.WAITING,
+            OrderStatus.OFFERED,
```

Comment:
Same as the comment on `leases/services/invoice.py`, let's set them to `DRAFTED` (and maybe even remove the due date?) and let the actual sending handle the status change to `OFFERED`.